EncounterMe/EncounterMe/Views/AllObjectsPage.xaml.cs

```diff
@@ -69,5 +73,37 @@ async void Add_Object_Button_Clicked(object sender, EventArgs e)
         {
             await PopupNavigation.Instance.PushAsync(new AddObjectPopup());
         }
+
+        async void listView_ItemSelected(object sender, ItemTappedEventArgs e)
+        {
+
+            var pinToPass = ((ListView)sender).SelectedItem as MapPin;
+            if (pinToPass == null)
+            {
+                return;
+            }
+
+            //await Shell.Current.DisplayAlert("Map Pin selected", pinToPass.Name, "okey");
+            await Shell.Current.Navigation.PushAsync(new IndividualObjectPage(pinToPass));
+
+
+        }
+
+        void listView_ItemTapped(object sender, ItemTappedEventArgs e)
+        {
+            ((ListView)sender).SelectedItem = null;
+        }
+
+       async void More_Info_Clicked(object sender, EventArgs e)
+        {
+            var pinToPass = ((MenuItem)sender).BindingContext as MapPin;
+            if(pinToPass == null)
```

Comment:
O įmanoma kaži kažkaip paspaust tą MoreInfo be jokio objekto? Nes gal nereikalingas toks tikrinimas

Reply:
Nelabai, nes jis atsiranda tik ilgiau paspaudus ant objekto.
Jei ir taip atsitiktų tai tas pin gautas būtų null ir return'intų iš metodo

Reply:
O tas more info ir ves i ta pacia vieta? ar bus kazkaip parodoma daugiau info apie kurejus objekto ir pns

Reply:
Kolkas tai nukreipia ten pat kaip ir tiesiog paspaudus ant objekto. Ateity galima būtų ir į kitą puslapį nukreipt

Reply:
> Nelabai, nes jis atsiranda tik ilgiau paspaudus ant objekto. Jei ir taip atsitiktų tai tas pin gautas būtų null ir return'intų iš metodo

Tai jei taip neįmanoma, gal nereik papildomų kodo eilučių su tuo tikrinimu, gal pristatant darbą be šito exceptionų neišmes :DDDDD

Reply:
Tas tikrinimas ar gautas pinToPass yra ne null tam nes čia keliaujam į tą patį puslapy kur aprašymas yra apie objektą. Jei jį išmesim, tai paduos į tą puslapį mapPin kuris yra null ir išmes exception 😀